packages/manager/modules/exchange/src/dashboard/translations/Messages_pt_PT.json

```diff
@@ -490,7 +490,7 @@
   "exchange_ACTION_add_no_domains": "Ainda não adicionou um domínio ou o mesmo ainda não foi validado <br/>\\ Deverá adicionar um domínio e aguardar pela sua validação para criar um alias para a sua conta Exchange. ",
   "exchange_ACTION_add_account_option_fail": "Ocorreu um erro aquando do carregamento dos seus domínios.",
   "exchange_ACTION_add_account_fetchingAccountCreationOptions_error": "Ocorreu um erro durante a obtenção das opções de criação de uma conta Exchange.",
-  "exchange_ACTION_add_account_error_message": "Ocorreu um erro aquando da criação da conta Exchange.",
+  "exchange_ACTION_add_account_error_message": "Ocorreu um erro aquando da criação da conta Exchange: {{message}}",
```

Comment:
**nitpick:** Shouldn't be done manually. Please only apply changes on `Messages_fr_FR.json` files.

Reply:
Ok, i will asked for translation then.

Reply:
Reverted for all languages except fr.